src/main/res/values/strings.xml

```diff
@@ -1303,17 +1303,17 @@
 	</plurals>
 
 	<!-- 2020-09-19 -->
-	<string name="action_save_image_success_no_path">Image saved successfully</string>
+	<string name="action_save_image_success_no_path">Image saved</string>
 
 	<!-- 2020-10-20 -->
 	<string name="pref_behaviour_imagevideo_tap_close_key" translatable="false">pref_behaviour_imagevideo_tap_close</string>
 	<string name="pref_behaviour_imagevideo_tap_close_title">Close images/videos on tap</string>
-	<string name="pref_behaviour_imagevideo_tap_close_summary">Does not affect videos if playback controls are enabled</string>
+	<string name="pref_behaviour_imagevideo_tap_close_summary">Does not affect videos if playback controls are on</string>
 
 	<!-- 2020-11-13 -->
 	<string name="press_back_again">Press back again</string>
 	<string name="pref_behaviour_postlist_back_again_key" translatable="false">pref_behaviour_back_again</string>
-	<string name="pref_behaviour_postlist_back_again">Press back twice to exit post list</string>
+	<string name="pref_behaviour_postlist_back_again">Press \'Back\' twice to exit the post list</string>
```

Comment:
Similar to other comment, the button isn't labelled "Back", so I'd prefer this as "Press back twice to exit the post list"

Reply:
Is it "←"?

Reply:
It's the Android system back button, which is normally a left-pointing triangle, but the actual look is system-dependent. This was fine as is.